c2corg_ui/static/js/documentediting.js

```diff
@@ -33,7 +33,14 @@ app.documentEditingDirective = function() {
     scope: true,
     controller: 'appDocumentEditingController',
     controllerAs: 'editCtrl',
-    bindToController: true
+    bindToController: true,
+    link: function(scope, el, attrs, ctrl) {
+      // The directive executes after the dom is loaded but we have to wait until the expressions are evaluated
```

Comment:
The number of steps has no connection with the API/DB. It is only related with the UI and the way it works. So it's not a  good idea to add it to the DB.
But it could be set in some config in the UI, for instance in https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/constants.js ?